src/main/java/io/xpring/codec/addresses/ClassicAddressCodec.java

```diff
@@ -0,0 +1,82 @@
+package io.xpring.codec.addresses;
+
+
+import io.xpring.codec.base58.B58;
+import io.xpring.codec.basex.EncodingFormatException;
+
+public class ClassicAddressCodec {
+  public static final B58 codec = new B58("rpshnaf39wBUDNEGHJKLM4PQRST7VWXYZ2bcdeCg65jkm8oFqi1tuvAxyz");
+
+  // Otherwise known as `family seed`
+  public static final B58.Version SEED_K256 = new B58.Version(
```

Comment:
Optional nit: you may refer to these as "prefixes" though I understand the library you are using uses "version". 

Either way, `SEED_K256_VERSION` or `SEED_K256_PREFIX` seem like more descriptive variable names without sacrificing too much verbosity.